### PR TITLE
Minor fixes to group, computer, pre-commit

### PR DESCRIPTION
**Pull request by ltalirz**
These are just a few minor improvements to `group`, `computer` and the `pre-commit` documentation.

### CODE REVIEW

File: aiida/orm/implementation/django/computer.py [hunk 1]
```diff
@@ -222,6 +222,9 @@ def get_workdir(self):
 
     def set_workdir(self, val):
         # if self.to_be_stored:
+        if not isinstance(val, basestring):
```
Comment (DropD) [inline aiida/orm/implementation/django/computer.py]: Consider using six.string_types (python 3 compatible), six is already an indirect dependency. May be too early to even think about that though

File: aiida/orm/implementation/general/group.py [hunk 2]
```diff
@@ -133,6 +133,18 @@ def get_db_columns():
         """
         pass
 
+    @classmethod
+    def create(cls, *args, **kwargs):
+        """
+        Create a new group.
```
Comment (DropD) [inline aiida/orm/implementation/general/group.py]: Nit: Should this mention that "create" includes storing in the db? The name is a documented django convention but do we stick to it / have it documented anywhere?

File: aiida/orm/implementation/django/computer.py (hunk 1 above)
Comment (ltalirz) [reply aiida/orm/implementation/django/computer.py]: I think this replacement should be practically automatic when we do the transition.
Let's do it when it becomes relevant.

File: aiida/orm/implementation/general/group.py (hunk 2 above)
Comment (ltalirz) [reply aiida/orm/implementation/general/group.py]: Updated help string to point out that it stores the group.

**Pull request merged**